src/lib/Tests/Form/DataTransformer/ContentInfoTransformerTest.php

```diff
@@ -16,30 +16,34 @@
 use eZ\Publish\API\Repository\Exceptions\NotFoundException;
 use PHPUnit\Framework\MockObject\MockObject;
 
-class ContentInfoTransformerTest extends TestCase
+final class ContentInfoTransformerTest extends TestCase
 {
-    /** @var ContentInfoTransformer */
+    private const EXAMPLE_CONTENT_ID = 123456;
+
+    /** @var \EzSystems\EzPlatformAdminUi\Form\DataTransformer\ContentInfoTransformer */
     private $contentInfoTransformer;
 
     protected function setUp(): void
     {
-        /** @var ContentService|MockObject $contentService */
+        /** @var \eZ\Publish\API\Repository\ContentService|\PHPUnit\Framework\MockObject\MockObject $contentService */
         $contentService = $this->createMock(ContentService::class);
-        $contentService->expects(self::any())
+        $contentService
             ->method('loadContentInfo')
-            ->with(123456)
-            ->willReturn(new ContentInfo(['id' => 123456]));
+            ->with($this->logicalAnd(
+                $this->equalTo(self::EXAMPLE_CONTENT_ID),
+                $this->isType('int')
+            ))
+            ->willReturn(new ContentInfo([
+                'id' => self::EXAMPLE_CONTENT_ID,
+            ]));
 
         $this->contentInfoTransformer = new ContentInfoTransformer($contentService);
     }
 
     /**
      * @dataProvider transformDataProvider
-     *
-     * @param $value
-     * @param $expected
      */
-    public function testTransform($value, $expected)
+    public function testTransform($value, $expected): void
```

Comment:
missing strict type

Reply:
Fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/1148/commits/42437d627f79cccd4225b508580a7512e00f0d8d